Settings.bundle/Root.plist

```diff
@@ -6,6 +6,30 @@
 	<string>Root</string>
 	<key>PreferenceSpecifiers</key>
 	<array>
+		<dict>
+			<key>Type</key>
+			<string>PSMultiValueSpecifier</string>
+			<key>Title</key>
+			<string>Event Search Radius</string>
+			<key>Key</key>
+			<string>searchRadiusSetting</string>
+			<key>DefaultValue</key>
+			<integer>1</integer>
+			<key>Titles</key>
+			<array>
+				<string>10</string>
```

Comment:
Just a number seems not 100% clear to me, maybe adding " km" after the number?

Reply:
When adding " km" to the titles entries, the feature still works and shows the additional information in the UI. :+1: What do you think @mltbnz ?

Reply:
think that makes sense. 
Since I saw that the API now also supports Event types (like Kidical Mass, Critical Mass and others) I thought to a selection screen for events you are interested in as well and then maybe it actually makes sense to move the option code somewhere inside the app. maybe a button on the map to configure the feature and maybe some people do not even want to see events on the map... 🤔 

WDYT?

Reply:
Yes, sounds like this setting should move inside the app. 🤔